mini_pupper_description/urdf/mini_pupper_2_description.urdf

```diff
@@ -0,0 +1,1287 @@
+      <material name="opaque2552210"/>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2_lpf1.stl" scale="1 1 1"/>
+      </geometry>
+    </collision>
+  </link>
+  <link name="lpb1">
+    <inertial>
+      <origin rpy="0 0 0" xyz="0.002259 0.008667 -0.002748"/>
+      <mass value="0.002574"/>
+      <inertia ixx="0.39059e-6" ixy="24.631e-9" ixz="54.116e-9" iyy="0.412444e-6" iyz="3.861e-9" izz="0.246337e-6"/>
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2_lpb1.stl" scale="1 1 1"/>
+      </geometry>
+      <material name="opaque2552210"/>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2_lpb1.stl" scale="1 1 1"/>
+      </geometry>
+    </collision>
+  </link>
+  <link name="rpf1">
+    <inertial>
+      <origin rpy="0 0 0" xyz="0.002259 -0.008667 -0.002748"/>
+      <mass value="0.002574"/>
+      <inertia ixx="0.39059e-6" ixy="-24.631e-9" ixz="54.116e-9" iyy="0.412444e-6" iyz="-3.861e-9" izz="0.246337e-6"/>
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2_rpf1.stl" scale="1 1 1"/>
+      </geometry>
+      <material name="opaque2552210"/>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2_rpf1.stl" scale="1 1 1"/>
+      </geometry>
+    </collision>
+  </link>
+  <link name="rpb1">
+    <inertial>
+      <origin rpy="0 0 0" xyz="0.002259 -0.008667 -0.002748"/>
+      <mass value="0.002574"/>
+      <inertia ixx="0.39059e-6" ixy="-24.631e-9" ixz="54.116e-9" iyy="0.412444e-6" iyz="-3.861e-9" izz="0.246337e-6"/>
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2_rpb1.stl" scale="1 1 1"/>
+      </geometry>
+      <material name="opaque2552210"/>
+    </visual>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2_rpb1.stl" scale="1 1 1"/>
+      </geometry>
+    </collision>
+  </link>
+  <link name="lpf2">
+    <inertial>
+      <origin rpy="0 0 0" xyz="-0.001747 0.007904 0.005365"/>
+      <mass value="0.002231"/>
+      <inertia ixx="0.862567e-6" ixy="1.254e-9" ixz="-0.112894e-6" iyy="0.991403e-6" iyz="-13.212e-9" izz="0.13904e-6"/>
+    </inertial>
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2_lpf2.stl" scale="1 1 1"/>
```

Comment:
```suggestion
        <mesh filename="package://mini_pupper_description/meshes/mini_pupper_2/lpf2.stl" scale="1 1 1"/>
```
I think make folders `meshes/mini_pupper_2/` and `meshes/mini_pupper/` is cleaner and more organised for the mesh files, also for the links.yaml, gait.yaml and joints.ymal